Makefile.containers

```diff
@@ -10,8 +10,8 @@ endif
 
 ifeq ($(CONTAINER_CLITOOL), docker)
 CONTAINER_BUILDENV ?= DOCKER_BUILDKIT=1 BUILDKIT_PROGRESS=plain
-ifeq ($(shell docker-compose -v >/dev/null 2>&1 || echo FAIL),)
-CONTAINER_COMPOSE = docker-compose
+ifeq ($(shell docker -v >/dev/null 2>&1 || echo FAIL),)
```

Comment:
This change might break for people running docker on mac os I think

Reply:
It shouldn't, docker-compose hasn't been updated since 2021. It comes out of the box with docker now. https://docs.docker.com/compose/install/